pkg/cloud/aws/aws.go

```diff
@@ -190,6 +157,7 @@ func (a *awsProvider) PrepareSubmarinerClusterEnv() error {
 // 4. revoke Submariner metrics ports
 // 5. revoke Submariner route ports
 func (a *awsProvider) CleanUpSubmarinerClusterEnv() error {
+	//a.cloudPrepare.CleanupAfterSubmariner(a)
```

Comment:
Why is this commented out?

Reply:
Because we need to use this for clean-up, but it currently blows up. I’ll remove it for now.